admin-dev/themes/new-theme/js/components/change-password-handler.js

```diff
@@ -103,7 +103,7 @@ export default class ChangePasswordHandler {
       case $.passy.strength.LOW:
         return {
           message: this.$feedbackContainer.find('.strength-low').text(),
-          elementClass: 'text-danger',
+          elementClass: 'text-warning',
         };
```

Comment:
Changing this because I am also changing low streingth warning from "invalid" to "weak". 

I think that makes more sense because even if password is "invalid" according to passy(the library responsible for password strength), it will still be saved. 
In general version of passy in PrestaShop is quite outdated and hardly makes sense ("123456789" is a "fabulous" password, but 0123456789 is "invalid") and needs to be changed/removed/updated but I think it's out of scope for this PR.